<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgWndAskPINClass</class>
 <widget class="QDialog" name="dlgWndAskPINClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dlgWndAskPIN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QFrame" name="fraPIN_Keypad">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>116</width>
     <height>177</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout">
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="lblPINName_2">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>PIN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPIN_2">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="tbtNUM_1">
      <property name="text">
       <string>&amp;1</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::DelayedPopup</enum>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonIconOnly</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="tbtNUM_2">
      <property name="text">
       <string>&amp;2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="tbtNUM_3">
      <property name="text">
       <string>&amp;3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="tbtNUM_4">
      <property name="text">
       <string>&amp;4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="tbtNUM_5">
      <property name="text">
       <string>&amp;5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="tbtNUM_6">
      <property name="text">
       <string>&amp;6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QToolButton" name="tbtNUM_7">
      <property name="text">
       <string>&amp;7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QToolButton" name="tbtNUM_8">
      <property name="text">
       <string>&amp;8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QToolButton" name="tbtNUM_9">
      <property name="text">
       <string>&amp;9</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QToolButton" name="tbtNUM_0">
      <property name="text">
       <string>&amp;0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QToolButton" name="tbtClear">
      <property name="text">
       <string>C&amp;E</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="fraPIN_Normal">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>71</width>
     <height>77</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <layout class="QGridLayout">
    <item row="0" column="1">
     <spacer>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::MinimumExpanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::MinimumExpanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <spacer>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblPINName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Resources/alpha.png);</string>
   </property>
   <property name="text">
    <string>PIN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPIN">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHeader">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Resources/alpha.png);</string>
   </property>
   <property name="text">
    <string>Enter you PIN</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>288</width>
     <height>68</height>
    </rect>
   </property>
   <layout class="QGridLayout">
    <item row="3" column="1">
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>271</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="lblIcon">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>dlgWndAskPINClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>dlgWndAskPINClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
